minifi-bootstrap/src/main/java/org/apache/nifi/minifi/bootstrap/RunMiNiFi.java

```diff
@@ -1379,7 +1430,7 @@ private void saveFile(final InputStream configInputStream, File configFile) {
             }
         }
 
-        private void performTransformation(InputStream configIs, String configDestinationPath) {
+        public void performTransformation(InputStream configIs, String configDestinationPath) {
```

Comment:
Understand the need for increased visibility but think protected or package private would be sufficient.